server/routes/profile.js

```diff
@@ -0,0 +1,72 @@
+const express = require('express')
+const router = express.Router()
+const Profile = require('../models/profileModel')
+
+// Add a new profile
+router.post('/profile', (req, res) => {
```

Comment:
So we probably will not be asking for all of this information on signup. We should probably handle what happens if some of these fields are not provided. I'm also thinking that we should alter the profile model because definitely not everything is required. The rest of the team can weigh in on this but I'm thinking the only things required should be firstName, lastName, and email. Things like address are then only required for stripe payments, and availability is only required for when you want to do dog sitting